assets/src/apps/delivery/store/features/groups/actions/sequence.ts

```diff
@@ -139,9 +139,17 @@ export const getSequenceLineage = (
   return lineage;
 };
 
-export const getIsLayer = () => {
+export const getSequenceInstance = () => {
   const currentSequenceId = useSelector(selectCurrentSequenceId);
```

Comment:
I didn't think it was possible to use hooks inside a "normal" method... I realize this wasn't you. making a mental note to refactor these 3 methods to take the sequenceId and sequence as input instead. should useSelector inside the component and pass to these methods (also makes them testable). I assume this must be working only because they are being called inside a component in the end? I can't imagine they are properly reactive however...